snapcraft/plugins/_python/_pip.py

```diff
@@ -159,20 +171,20 @@ def _ensure_pip_installed(self):
         if not self._is_pip_installed():
             logger.info('Fetching and installing pip...')
 
-            real_python_home = self._python_home
+            real_python_home = self.__python_home
```

Comment:
doesn't this have an extra _?

Reply:
ah, this is weird :-P

Reply:
Yeah it looks a tad off, but it maintains current behavior: "Ignore whatever you think you know, use this one. Okay, now you can go back to normal."

Reply:
Yeah, it is made clear in the comments below, but not as I read line by line until I reached the end of it :smile: